cs/src/comm/epoxy-transport/EpoxyListener.cs

```diff
@@ -5,32 +5,45 @@ namespace Bond.Comm.Epoxy
 {
     using System;
     using System.Collections.Generic;
+    using System.Diagnostics;
     using System.Net;
     using System.Net.Sockets;
+    using System.Security.Authentication;
     using System.Threading;
     using System.Threading.Tasks;
     using Bond.Comm.Service;
 
     public class EpoxyListener : Listener
     {
-        private EpoxyTransport parentTransport;
-        private System.Net.Sockets.TcpListener listener;
-        private ServiceHost serviceHost;
+        EpoxyTransport parentTransport;
+        EpoxyServerTlsConfig tlsConfig;
+        System.Net.Sockets.TcpListener listener;
```

Comment:
Not anymore. It was needed before when `EpoxyListener` was called `TcpListener`.